File: SampleApp/Objective-C/SampleApp/BannerNavigationController.m
//
//  BannerNavigationController.m
//  SampleApp
//
//  Fluct SDK
//  Copyright © 2016年 fluct, Inc. All rights reserved.
//

#import "BannerNavigationController.h"
#import <FluctSDK/FluctSDK.h>

@interface BannerNavigationController ()

@end

@implementation BannerNavigationController

- (void)viewDidLoad {
    [super viewDidLoad];
    FSSBannerView *bannerView = [[FSSBannerView alloc] initWithFrame:CGRectMake(0, CGRectGetHeight(self.view.frame) - 100, 320, 50)];
    [bannerView setRootViewController:self];
    [bannerView setMediaID:@"0000005617"];
    [self.view addSubview:bannerView];
}

@end

File: SampleApp/Swift/SampleApp-Swift/BannerNavigationController.swift
//
//  BannerNavigationController.swift
//  SampleApp-Swift
//
//  Fluct SDK
//  Copyright © 2016年 fluct, Inc. All rights reserved.
//

import UIKit
import FluctSDK

class BannerNavigationController: UINavigationController {

    override func viewDidLoad() {
        super.viewDidLoad()
        let bannerView = FSSBannerView(frame: CGRect(x: 0, y: self.view.frame.height - 100, width: 320, height: 50))
        bannerView.setMediaID("0000005617")
        bannerView.setRootViewController(self)
        self.view.addSubview(bannerView)
    }
}
